src/main/scala/io/iohk/ethereum/consensus/ethash/difficulty/EthashDifficultyCalculator.scala

```diff
@@ -15,9 +18,15 @@ class EthashDifficultyCalculator(blockchainConfig: BlockchainConfig) extends Dif
   val ConstantinopleRelaxDifficulty: BigInt = 5000000
 
   def calculateDifficulty(blockNumber: BigInt, blockTimestamp: Long, parentHeader: BlockHeader): BigInt = {
-    lazy val timestampDiff = blockTimestamp - parentHeader.unixTimestamp
+    lazy val GrandparentsData(grandparentDifficulty, grandparentsTimestampDiff) = grandParentsDataGetter(
```

Comment:
Ok, my OCD is breaking through the roof, but I would put everything after `=` on a single new line